schemas/openshift/terraform-resource-1.yml

```diff
@@ -813,6 +815,10 @@ oneOf:
       "$ref": "/aws/tenant_accounts-1.yml#/properties/account"
     identifier:
       type: string
+    name:
+      description: base fqdn of the zone
+      type: string
+      format: uri
```

Comment:
no, uris are more than we need

This should be better https://github.com/app-sre/qontract-schemas/blob/main/schemas/common-1.json#L47-L51 since k8s container names are valid DNS names

Reply:
done in https://github.com/app-sre/qontract-schemas/pull/85/commits/1569085eddfa124dfa752b37818ebf4760244dd4